assets/locales/lv.json

```diff
@@ -45,7 +45,8 @@
     "thisIsRequired": "Šis ir nepieciešams!",
     "tileSize": "\"Flīžu\" izmērs",
     "useProxy": "Izmantojiet starpniekserveri",
-    "useTiles": "Izmantojiet \"flīzes\""
+    "useTiles": "Izmantojiet \"flīzes\"",
+    "addSelected": "Add selected layers to the map"
```

Comment:
@DailisLangovskis @raitisbe Perhaps one of you could fill in the translation before merge?

Reply:
@jmacura done